scheduler/scheduler.cpp

```diff
@@ -1766,16 +1770,19 @@ static int prepare_broadcast_reply(char* buf, const char* netname)
     }
 }
 
-static void broadcast_scheduler_version()
+static void broadcast_scheduler_version(const char* netname)
 {
-    const int schedbuflen = 4 + sizeof(uint64_t);
+    char length_netname = sizeof(netname);
+    const int schedbuflen = 5 + sizeof(uint64_t) + length_netname;
     char buf[ schedbuflen ];
```

Comment:
you have to use new/delete here until we have update to C++11.  I want to get this release out with C++98 before forcing the upgrade.  This only compiles now because length_netname is a compiletime constant, but that isn't what you want it to be.